app/Libraries/OsuAuthorize.php

```diff
@@ -374,6 +374,16 @@ public function checkBeatmapsetNominate($user, $beatmapset)
             return $prefix.'owner';
         }
 
+        if ($user->isProbationaryBN()) {
```

Comment:
This should be checking if a user has either the qat or full bn usergroup like line 437.
A full standard bn that is in probation for mania should be allowed to nominate a standard/taiko hybrid.

Reply:
there aren't mode-specific usergroups for BN/probation so for the purpose of the website your example user should just be a full BN

otherwise this works as intended